dist/index.html

```diff
@@ -3,11 +3,11 @@
 <html lang="en">
 <head>
   <meta charset="UTF-8">
-  <title>Tidal Migrations API</title>
+  <title>Tidal Migrations API <img src="tidal_icon.png" alt="Tidal Migrations logo"> </title>
```

Comment:
oh does this work? does it put the logo _after_ the title in the tab header?

Reply:
this was by accident, ill remove the changes for this file